.pre-commit-config.yaml

```diff
@@ -50,3 +50,7 @@ repos:
       -D,
       warnings,
     ]
+- repo: https://github.com/Holzhaus/sourceheaders
+  rev: 41391140ca6ac7c3dfddae090353d183cbeadb48
+  hooks:
+  - id: sourceheaders
```

Comment:
would've wished this was written in Rust (cuz u know, [oxidize it](https://wiki.mozilla.org/Oxidation)) but python makes sense too for pre-commit. ;)

Reply:
I thought about it, but unfortunately rust code requires a system install of rust on the host, so I decided against it.